components/brave_rewards/browser/rewards_service_impl.h

```diff
@@ -247,6 +256,10 @@ class RewardsServiceImpl : public RewardsService,
   void MaybeShowBackupNotification(uint64_t boot_stamp);
   void MaybeShowAddFundsNotification(uint64_t reconcile_stamp);
 
+  void OnRemovePendingContribution(bool result);
```

Comment:
OnPendingContributionRemoved to be consistent with other method names

Reply:
fixed